src/platform-includes/enriching-events/attach-viewhierarchy/android.mdx

```diff
@@ -3,3 +3,25 @@
   <meta-data android:name="io.sentry.attach-view-hierarchy" android:value="true" />
 </application>
 ```
+
+### Jetpack Compose Support using the Sentry Kotlin Compiler Plugin
+
+<Note>
+
+The Sentry Kotlin Compiler plugin is considered _experimental_. Give it a try and provide early feedback [on GitHub](https://github.com/getsentry/sentry-android-gradle-plugin).
+
+</Note>
+
+If your application uses Jetpack Compose, you can use the <PlatformLink to="/kotlin-compiler-plugin/">Sentry Kotlin Compiler Plugin</PlatformLink> to see the function names of your `@Composable` elements. You must add the plugin to all modules that contain `@Composable` elements.
+
+```groovy {filename:app/build.gradle}
+plugins {
+  id "io.sentry.kotlin.compiler.gradle" version "{{@inject packages.version('sentry.kotlin.compiler-plugin', '3.0.0') }}"
```

Comment:
Forgot to say that, but we should specify the min version where we started publishing the compiler plugin (3.7.0 in our case, that's where we added the plugin marker file). In case docs fail to pull the version from registry, at least it would show the correct min version. This has to be updated in other places in this PR as well
```suggestion
  id "io.sentry.kotlin.compiler.gradle" version "{{@inject packages.version('sentry.kotlin.compiler-plugin', '3.7.0') }}"
```